ext/drwrap/drwrap.c

```diff
@@ -981,6 +981,11 @@ drwrap_exit(void)
         !dr_unregister_delete_event(drwrap_fragment_delete))
         ASSERT(false, "failed to unregister in drwrap_exit");
 
+    for (i = 0; i < POSTCALL_CACHE_SIZE; i++) {
+        postcall_cache[i] = NULL;
+    }
```

Comment:
This is fine for this CL but it brings up two thoughts:

1) For large data structures zeroing at exit is costly when there's no chance or re-attach: which is all of the time for regular non-static DR.  Inside DR we only zero for doing_detach.  We may want to export that for extensions.

2) We should add a re-attach test for drwrap.

I filed #3065 for this, split from #2157.  Let's reference both issues in the merge commit message.

Reply:
Sure